java/dagger/Module.java

```diff
@@ -32,7 +32,7 @@
    * Additional {@code @Module}-annotated classes from which this module is
    * composed. The de-duplicated contributions of the modules in
    * {@code includes}, and of their inclusions recursively, are all contributed
-   * to the object graph.
+   * to the component.
```

Comment:
This is actually in a few places. If this is worth doing here, did you want to nip the other four places?